sdk/openai/openai-assistants/review/openai-assistants-api.api.md

```diff
@@ -41,7 +41,7 @@ export function createMessage(context: AssistantsContext, threadId: string, role
 export function createRun(context: AssistantsContext, threadId: string, createRunOptions: CreateRunOptions, options?: CreateRunRequestOptions): Promise<ThreadRun>;
 
 // @public
-export function createThread(context: AssistantsContext, body: AssistantThreadCreationOptions, options?: CreateThreadOptions): Promise<AssistantThread>;
+export function createThread(context: AssistantsContext, body: AssistantThreadCreationOptions_2, options?: CreateThreadOptions): Promise<AssistantThread>;
```

Comment:
typo?

Reply:
hi @glharper, no, the `_2` suffix is auto-generated when run `rush build`. I think it came from the fact that the name is used in both `api/*` and `rest/*` models.

You'll find a similar `_2` suffix [a few lines above](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/openai/openai-assistants/review/openai-assistants-api.api.md?plain=1#L29).